### PR TITLE
Add t.Helper() call to all functions and bump Go version to 1.9+

### PR DESCRIPTION
**Pull request by eugene-gurevich**
The errors will now output the calling file:line, rather than the internal location of t.Error.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 language: go
 
 go:
-            - 1.7
+            - 1.9
```
Comment (theothertomelliott) [inline .travis.yml]: You can specify multiple targets. I'd suggest including 1.10 and 1.11 as well.

File: must.go [hunk 2]
```diff
@@ -3,11 +3,13 @@ package must
...
 type TestingT interface {
 	Errorf(format string, args ...interface{})
+	Helper()
 }
 
 // MustTester defines an interface with functions matching the package level check functions, without the requirement to specify a TestingT.
 type MustTester interface {
 	BeEqual(expected, got interface{}, a ...interface{}) bool
 	BeEqualErrors(expected, got error, a ...interface{}) bool
+	BeNoError(got error, a ...interface{}) bool
```
Comment (theothertomelliott) [inline must.go]: Weren't you planning on allowing a bool to be passed here to verify whether or not an error was expected?

File: .travis.yml (hunk 1 above)
Comment (theothertomelliott) [reply .travis.yml]: NB: 1.10 needs to be specified as a string.
https://docs.travis-ci.com/user/languages/go/#specifying-a-go-version-to-use

Comment (eugene-gurevich) [reply .travis.yml]: Done

File: must.go (hunk 2 above)
Comment (eugene-gurevich) [reply must.go]: That's coming up. This was just adding the already existing function to the interface.

**Comment by eugene-gurevich** [comment]
Addressed the comments 
(I'm not sure whether you see my responses, though?)

File: must.go (hunk 2 above)
Comment (theothertomelliott) [reply must.go]: Awesome, thanks for catching the missing item!

**Comment by theothertomelliott** [comment]
Yep, saw your comments. Got separate mails for them, but saw the comments nontheless :P

**Pull request merged**